pyproject.toml

```diff
@@ -32,6 +32,13 @@ pytest-cov = "^3.0.0"
 pytest-mock = "^3.6.1"
 mypy = "*"
 
+[tool.poetry.scripts]
+export = 'topo_processor.cli.geostore.export:main'
+status = 'topo_processor.cli.geostore.status:main'
+list = 'topo_processor.cli.geostore.list:main'
+delete = 'topo_processor.cli.geostore.delete:main'
+validate = 'topo_processor.cli.validate:main'
+
```

Comment:
I'm not generally a fan of typing more than necessary but I wonder if it would be clearer if the commands were prefixed by `geostore-` (or something else more identifiable/less generic)?